src/html/components/buttons.html

```diff
@@ -0,0 +1,22 @@
+<h2>Buttons</h2>
+
+<a class="Btn Btn-Primary" href="#">Primary Button</a>
+<a class="Btn Btn-Secondary" href="#">Secondary Button</a>
+<a class="Btn Btn-Tertiary" href="#">Tertiary Button</a>
+
+<br/ >
+
+<input type="submit" class="Btn Btn-Primary" value="Primary Button" />
+<button class="Btn Btn-Secondary">Secondary Button</button>
+<a class="Btn Btn-Transparent" href="#">Plain button</a>
```

Comment:
@mattheu You should have an email with a comment I made here! Please can you paste in?